src/Type/Definition/ObjectType.php

```diff
@@ -145,7 +145,7 @@ public function getInterfaces()
             foreach ($interfaces as $iface) {
                 $iface = Type::resolve($iface);
                 if (!$iface instanceof InterfaceType) {
-                    throw new InvariantViolation("Expecting interface type, got " . Utils::printSafe($iface));
+                    throw new UserError(sprintf('Expecting interface type, got %s', Utils::printSafe($iface)));
```

Comment:
This looks like a perfect InvariantViolationError for me, as it's server developer error (in type definition). Is there any reason why this must be UserError?